src/main/java/org/folio/rest/impl/InitConfigService.java

```diff
@@ -15,7 +15,6 @@
   @Override
   public void init(Vertx vertx, Context context, Handler<AsyncResult<Boolean>> handler) {
     logger.info("Init Config Service");
-    logger.info("Loading default mappings from: " + MappingHelper.PATH);
```

Comment:
Do we still want to try to at least read the files in at startup so that it's immediately clear that something is wrong instead of waiting unil they're needed?

Reply:
Do we want to read the files? or just check if the files exists?

Reply:
Might as well read them too...